config_manager.py

```
@@ -57,19 +69,41 @@ class ConfigManager:
         
         try:
             with open(self.config_path, 'r', encoding='utf-8') as f:
                 return yaml.safe_load(f) or {}
         except yaml.YAMLError as e:
+            error = ConfigError(
+                message=f"設定ファイルの読み込みに失敗しました: {e}",
+                severity=ErrorSeverity.CRITICAL,
+                original_exception=e
+            )
+            self.error_handler.handle_error(error)
             raise ConfigValidationError(f"設定ファイルの読み込みに失敗しました: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)